nuxeo-coldstorage-package/src/main/assemble/assembly.xml

```diff
@@ -17,6 +17,8 @@
       <fileset dir="src/main/resources" />
       <filterset>
         <filter token="VERSION" value="${maven.project.version}" />
+        <filter token="DISTRIB_VERSION" value="${nuxeo.platform.version}" />
+        <filter token="NUXEO_TARGET_VERSION" value="${nuxeo.target.version}" />
       </filterset>
     </copy>
```

Comment:
On this file, we should think if we want to declare dependencies (WebUI as a mandatory/optional?)
Same for the other addons, since all of them don't work without Web UI.